src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -409,7 +409,8 @@ public void buttonCvrFilePathClicked() {
     FileChooser fc = new FileChooser();
     fc.setInitialDirectory(new File(FileUtils.getUserDirectory()));
     fc.getExtensionFilters()
-        .add(new ExtensionFilter("Excel and JSON files", "*.xls", "*.xlsx", "*.json"));
+        .add(new ExtensionFilter("Excel, JSON, and XML files", "*.xls", "*.xlsx", "*.json",
```

Comment:
I don't think this is WAI because it seems like for Hart you don't specify an xml file, but a directory instead (that's what you've done in your test files). That's currently not possible if you do it through the GUI since it's a file chooser, not a directory chooser.

Reply:
Yes, that's true.  I added this when I was first building the reader by reading one file at a time.  I'll revert this - UI changes should be handled in a different PR: https://github.com/BrightSpots/rcv/issues/460